README.md

```diff
@@ -185,6 +185,9 @@ implementers may get useful functionality into the hands of developers sooner wh
 
 ## Notes on the polyfill
 
+The `dist` directory contains two versions of the polyfill. `inert.js` is the debug version, containing an inlined sourcemap, `inert.min.js` is
```

Comment:
Mention that these are both generated from the ES6 source, which can be used directly if desired?